components/alert/test/alert.test.js

```diff
@@ -59,11 +59,63 @@ describe('d2l-alert', () => {
 		it('calling preventDefault on close action should prevent alert from closing', async() => {
 			const alert = await fixture(alertFixture);
 			const closeButton = alert.shadowRoot.querySelector('d2l-button-icon');
-			setTimeout(() => closeButton.click());
 			alert.addEventListener('d2l-alert-close', (e) => {
 				e.preventDefault();
 			});
+			setTimeout(() => closeButton.click());
+			await oneEvent(alert, 'd2l-alert-close');
+			expect(alert.hasAttribute('hidden')).to.be.false;
+		});
+
+		it('preventing first close keeps alert visible; second close hides it', async() => {
```

Comment:
I don't know the that remainder of the tests really add much value (e.g., we're already testing preventDefault in the test above) but let me know if you have different thoughts.